bean.js

```diff
@@ -0,0 +1,13 @@
+const countBs = (string) => {
+    return countChar(string, "B")
+}
+
+let countChar = (string, char) => {
```

Comment:
If you are not reassigning anything it's better to use `const`, this way you are signaling to the reader, that the value won't be changed during program execution